turbo.json

```diff
@@ -2,6 +2,9 @@
   "$schema": "https://turbo.build/schema.json",
   "ui": "tui",
   "tasks": {
+    "build-wasm": {
+      "outputs": ["packages/account-wasm/pkg/**"]
+    },
```

Comment:
It's prob simpler to rename the task to build:deps in `packages/account-wasm/package.json` so `pnpm turbo build:deps` handles the all lib build

Reply:
we probably don't want to build wasm everywhere though as it has more deps and requires more resources

Reply:
ah I see, I guess that's why `pkg` is committed. Makes sense